### PR TITLE
Improve .snupkg documentation

### PR DESCRIPTION
**Pull request by loic-sharma**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[69b98bb](https://github.com/NuGet/docs.microsoft.com-nuget/commits/69b98bbf6a947b6fe0835c1c75635b4c7bb6251d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Symbol-Packages-snupkg.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-4/docs/create-packages/Symbol-Packages-snupkg.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg?branch=pr-en-us-1854) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C24%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202001242039524802-1854%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=pAWvUdSuJiDzNb9DmHGxCpfag8nyQmSaZmwSubtU14c%3D&st=2020-01-24T20%3A35%3A29Z&se=2020-02-24T20%3A40%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/create-packages/Symbol-Packages-snupkg.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ ms.reviewer:
 
 # Creating symbol packages (.snupkg)
 
-Symbol packages allow you to improve the debugging experience of your NuGet packages.
+A good debugging experience relies on the presence of debug symbols as they provide critical information like the association between the compiled and the source code, names of local variables, stack traces, and more. By default, these symbols are are built as separate files to minimize the size of shipping binaries. You can use symbol packages (.snupkg) to distribute these symbols and improve the debugging experience of your NuGet packages.
```
Comment (karann-msft) [inline docs/create-packages/Symbol-Packages-snupkg.md]: should we also call - to minimize the size of the nupkg?

File: docs/create-packages/Symbol-Packages-snupkg.md [hunk 2]
```diff
@@ -103,17 +103,17 @@ Symbol packages published to NuGet.org will fail validation if these constraints
 
 Symbol packages published to [NuGet.org](https://www.nuget.org/) undergo several validations, including malware scanning. If a package fails a validation check, its package details page will display an error message. In addition, the package's owners will receive an email with instructions on how to fix the identified issues.
 
-When the symbol package has passed all validations, the symbols will be indexed by NuGet.org's symbol servers. Once indexed, the symbol will be available for consumption from the NuGet.org symbol servers.
+When the symbol package has passed all validations, the symbols will be indexed by NuGet.org's symbol servers and will available for consumption.
```
Comment (karann-msft) [inline docs/create-packages/Symbol-Packages-snupkg.md]:
```suggestion
When the symbol package has passed all validations, the symbols will be indexed by NuGet.org's symbol servers and will be available for consumption.
```

Comment (loic-sharma) [reply docs/create-packages/Symbol-Packages-snupkg.md]: Nice catch 👍

Comment (loic-sharma) [reply docs/create-packages/Symbol-Packages-snupkg.md]: Nice catch 👍

File: docs/create-packages/Symbol-Packages-snupkg.md (hunk 1 above)
Comment (loic-sharma) [reply docs/create-packages/Symbol-Packages-snupkg.md]: Yeah that’s probably more important than explaining non-embedded PDBs. I change it up.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[4dfd63d](https://github.com/NuGet/docs.microsoft.com-nuget/commits/4dfd63db8763847a82e07c5949576723a1c00068)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Symbol-Packages-snupkg.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-4/docs/create-packages/Symbol-Packages-snupkg.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg?branch=pr-en-us-1854) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C31%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202001310100339984-1854%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=AyuuwZW5ljKcdr6u%2FGCfLdTpjc5gJzcxjcIr1%2FMg7lk%3D&st=2020-01-31T00%3A56%3A15Z&se=2020-03-02T01%3A01%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/create-packages/Symbol-Packages-snupkg.md (hunk 1 above)
Comment (loic-sharma) [reply docs/create-packages/Symbol-Packages-snupkg.md]: I ended up removing the comment as it felt like a little too low level a detail for an introduction of symbol packages.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[d62215d](https://github.com/NuGet/docs.microsoft.com-nuget/commits/d62215d354548899ffe8cf2651f6004cd1f5ae06)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Symbol-Packages-snupkg.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-4/docs/create-packages/Symbol-Packages-snupkg.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg?branch=pr-en-us-1854) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C31%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202001310100510830-1854%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=gdeC1bFmke3PvoGUEG1V9w3VMzJ7tzYHr88HrNU0zOk%3D&st=2020-01-31T00%3A57%3A01Z&se=2020-03-02T01%3A02%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**